packages/@aws-cdk/core/test/secret-value.test.ts

```diff
@@ -27,18 +27,21 @@ nodeunitShim({
 
     // THEN
     test.deepEqual(stack.resolve(v), '{{resolve:secretsmanager:secret-id:SecretString:json-key:version-stage:version-id}}');
+    test.equal(v.secretQualifier, { secretId: 'secret-id', jsonField: 'json-key', versionStage: 'version-stage', versionId: 'version-id' });
     test.done();
   },
 
   'secretsManager with defaults'(test: Test) {
     // GIVEN
     const stack = new Stack();
+    const secretId = 'secret-id';
 
     // WHEN
-    const v = SecretValue.secretsManager('secret-id');
+    const v = SecretValue.secretsManager(secretId);
 
     // THEN
     test.deepEqual(stack.resolve(v), '{{resolve:secretsmanager:secret-id:SecretString:::}}');
+    test.equal( v.secretQualifier, { secretId: secretId });
```

Comment:
Formatting is a little bit off:

```suggestion
    test.equal(v.secretQualifier, { secretId: secretId });
```